phpBB/styles/prosilver/template/ucp_agreement.html

```diff
@@ -62,7 +62,7 @@ <h2 class="sitename-title">{SITENAME} - {L_REGISTRATION}</h2>
 		<div class="inner">
 		<div class="content">
 			<h2 class="sitename-title">{SITENAME} - {AGREEMENT_TITLE}</h2>
-			<p>{AGREEMENT_TEXT}</p>
+			<div class="agreement">{AGREEMENT_TEXT}</div>
```

Comment:
put the <p class="agreement-text"> from above here

Reply:
@hanakin your above comment is not clear, could you please edit it?

Reply:
thx

Reply:
sorry this was a mistake I misread, but if we are changing a line then we should upgrade it to TWIG syntax